doc/classes/ProjectSettings.xml

```diff
@@ -2890,11 +2890,11 @@
 			If [code]true[/code], forces vertex shading for all rendering. This can increase performance a lot, but also reduces quality immensely. Can be used to optimize performance on low-end mobile devices.
 		</member>
 		<member name="rendering/textures/canvas_textures/default_texture_filter" type="int" setter="" getter="" default="1">
-			The default texture filtering mode to use on [CanvasItem]s.
+			The default texture filtering mode to use for [CanvasItem]s built-in texture. In shaders, this texture is always accessed as [code]TEXTURE[/code].
```

Comment:
```suggestion
			The default texture filtering mode to use for [CanvasItem]s built-in texture. In shaders, this texture is accessed as [code]TEXTURE[/code].
```
This texture can also be accessed by reading `COLOR` in a fragment shader, which does a texture read between the vertex and fragment functions, but doesn't require using the `TEXTURE` built-in. I don't think that fact needs to be mentioned here, necessarily, but "always accessed as" is technically a bit incorrect.

Reply:
Makes sense!